CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Improved grammar of microbreak ideas
 - display checkbox list vertically in settings screen 3
 - Dutch translations updated
+- Italian translations updated
```

Comment:
You should create new Updated part under Unreleased.

Reply:
Indeed! Fixed

Reply:
Should I have used a subsection like `### Changed`?